hw/ip/lc_ctrl/doc/_index.md

```diff
@@ -25,7 +25,7 @@ The life cycle controller provides the following features:
 - A CSR and a JTAG interface for initiating life cycle transitions.
 - Dedicated concurrent decoding of the redundant life cycle state and broadcasting of redundantly encoded life cycle qualification signals (e.g., to enable DFT features or the main processor).
 - A token hashing and matching mechanism to guard important life cycle transitions.
-- An escalation receiver for the alert subsystem, which allows to invalidate the life cycle state as part of an escalation sequence (see also [alert handler subsystem]({{< relref "hw/ip/alert_handler/doc" >}})).
+- An escalation receiver for the alert subsystem, which allows to invalidate the life cycle state as part of an escalation sequence (see also alert handler subsystem).
```

Comment:
for these paths, should we now be pointing to `hw/ip_autogen/alert_handler/doc`?

Reply:
Hm... I think I am with Tim on this one. It's true that there is potential for confusion - but everything in the docs is about `top_earlgrey` at the moment. 

I'd rather leave the links in, since it will be more difficult to restore this information at a later point otherwise.
I.e., leaving them there makes it easier to search and replace them with a potential solution for 3. in the future.

If this really turns out to get too confusing before we have a solution for 3., removing them should be straightforward to do.